lib/site.taxonomy.term.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Term class
+ */
+
+class Term {
+	/**
+	 * Create a Term instance
+	 *
+	 * @param {String} siteId - site id
+	 * @param {String} taxonomy - taxonomy type
+	 * @param {WPCOM} wpcom - wpcom instance
+	 * @return {Null} null
+	 */
+	constructor( siteId, taxonomy, term, wpcom ) {
```

Comment:
similar to `Taxonomy` class, to try to be consistent we should try to pass `term`, `taxonomy`, `siteId` and finally the `wpcom` instance.